setup.py

```diff
@@ -37,7 +37,7 @@
         "Programming Language :: Python :: 3.10",
     ],
     packages=find_packages(),
-    python_requires=">=3.10, <3.11",
+    python_requires=">=3.10, <3.12",
```

Comment:
Can we leave this out for now? We don't have a platform to test Python 3.11 right now. I agree we should make the switch soon, but I want to have some test plan built out first just in case.

Reply:
Agreed